src/status_im2/contexts/quo_preview/inputs/title_input.cljs

```diff
@@ -0,0 +1,54 @@
+(ns status-im2.contexts.quo-preview.inputs.title-input
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [quo2.core :as quo]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label "disabled?"
+    :key   :disabled?
+    :type  :boolean}
+   {:label "blur?"
+    :key   :blur?
+    :type  :boolean}
+  ])
+
+(defn cool-preview
+  []
+  (let [state          (reagent/atom {:blur?     false
+                                      :disabled? false})
+        value          (reagent/atom "")
+        max-length     24
+        on-change-text (fn [v]
+                         (reset! value v)
+                         (reagent/flush)
+                       )]
+    (fn []
+      [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
+       [rn/view {:padding-bottom 150}
+        [rn/view {:flex 1}]
+        [preview/customizer state descriptor]
+        [rn/view
+         {:padding-vertical  60
+          :flex-direction    :row
+          :margin-horizontal 20
+          :justify-content   :center}
+         [quo/title-input
+          {:max-length     max-length
+           :on-change-text on-change-text
+           :disabled?      (:disabled? @state)
+           :blur?          (:blur? @state)
+           :placeholder    "type something here"} @value]]]])))
```

Comment:
i mean here, we shouldn't pass @value

Reply:
hmm, that seems strange to me. is there not times then when the ui and data is out of sync?

Reply:
> hmm, that seems strange to me. is there not times then when the ui and data is out of sync?

why strange? why would you want to set value ? it was already set when you pressed the key

Reply:
I agree with @flexsurfer 

Just to make more clear the explanation:
```clojure
[my-text-component
 {:on-change     #(reset! @value,,,) ;; <- or any other function you want
  :default-value @value ;; <- this could be useful to initialize the field
                        ;; with a specified value, but it's not required
  :value         @value ;; <- you set this only if you manually want to update
                        ;; component's value. e.g. you have a "clear" button that
                        ;; when pressed it resets the atom to "", if you don't
                        ;; specify `:value @value`, then your change will not be
                        ;; reflected on the input.
  }]
```